### ISSUE TITLE
Uso de skills.json

### ISSUE THREAD

**Issue by notyourlasagna** [franciscogazitua/IIC2113-2024#143]
Buenas tardes, quería preguntar si el json skills es para informarnos a nosotros sobre las habilidades que existen solamente o si debemos además cargarlo en nuestro juego. Pregunto porque el json solo tiene nombre y descripción de la skill en lenguaje castellano, no es como que me sirva para cargar las clases de manera automática (creo). Se puede hacer el juego sin que cargue el json como tal? tipo haciendo una clase para cada skill según lo que hace y luego viendo cuáles tienen las units para el combate. En caso de no poderse, para qué es el skills.json? cómo lo debemos usar?

**Comment by RodrigoToroIcarte** [comment]
Efectivamente, no hay necesidad de leer el skills.json. Es más que nada información sobre las habilidades que podrías utilizar (si quieres) como referencia. Yo lo usé para algo muy específico en la E2. Cuando llegue ese momento te puedo contar para qué lo usé :)

**Comment by notyourlasagna** [comment]
Hola profe! Ya estoy haciendo la E2 (la segunda entrega que publicaron hoy) y por eso me nació la pregunta, así que feliz de saber en ese caso para qué lo usó :)) es necesario o se puede hacer la E2 y el resto del juego sin el json? porque me imagino que debo hacer una clase para cada habilidad y ahí no lo necesitaría más o sí? o sirve para ver las habilidades oficiales del juego por ejemplo por si algun equipo trae alguna habilidad inventada en sus units? no tengo claro si los testcases pueden traer info inventada ahí. no se me ocurre para qué más podría usar el json

**Comment by RodrigoToroIcarte** [comment]
Yo usé el skills.json para hacer un código en python que me imprima el esqueleto de mi factory de habilidades. Pasa que crear el factory con las habilidades es súper latero... hay que ir habilidad por habilidad poniendo el nombre correcto de cada habilidad y luego creando el objeto correspondiente.

Entonces yo hice un código que me generaba la fáctory (con los nombres de las habilidades sacados desde el json) y me dejara con un comentario la descripción de la habilidad. Así después me era fácil ir creando los objetos en mi factory de solo leer el comentario con lo que hace la habilidad. 

... pero fuera de eso, no hay ningún motivo para leer el skills.json desde tu proyecto. Ni en esta entrega ni en las siguientes.

**Comment by notyourlasagna** [comment]
excelente profesor muchísimas gracias! desafortunadamente creo que tomaré el camino latero a mano hehe
